Week1/js-exercises/ex4-whatsTheTime.js

```diff
@@ -13,6 +13,8 @@
 
 function displayCurrentTime() {
   // your code goes in here
+    const currentTime = new Date().toLocaleTimeString();
```

Comment:
This will show the wrong format for people in the UK and US.
```suggestion
    const currentTime = new Date().toLocaleTimeString('nl-nl');
```